fenicsxconcrete/experimental_setup/steel_beam_experiment.py

```diff
@@ -0,0 +1,87 @@
+from fenicsxconcrete.experimental_setup.base_experiment import Experiment
+from fenicsxconcrete.helpers import Parameters
+import dolfinx as df
+from mpi4py import MPI
+import numpy as np
+import ufl
+from petsc4py.PETSc import ScalarType
+
+class steel_beam(Experiment):
```

Comment:
I would  suggest to capitalize class names. This helps to differentiate between class and e.g. function names.